cmd/execution_builder.go

```diff
@@ -1126,15 +1127,23 @@ func (exeNode *ExecutionNode) LoadBootstrapper(node *NodeConfig) error {
 
 	// if the execution database does not exist, then we need to bootstrap the execution database.
 	if !bootstrapped {
+		err := wal.CheckpointHasRootHash(
+			node.Logger,
+			node.BootstrapDir,
```

Comment:
this expects the file to be in `/bootstrap/root.checkpoint`, but in `copyBootstrapState()`, it seems to expect it to be in `/boostrap/execution-state/root.checkpoint`

should this be?
```suggestion
			path.Join(node.BootstrapDir, bootstrapFilenames.DirnameExecutionState),
```